tensorflow_addons/layers/python/normalizations_test.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from tensorflow_addons.layers.python.normalizations import GroupNormalization,LayerNormalization,InstanceNormalization
+import numpy as np
+import scipy as scipy
+import tensorflow as tf
+from tensorflow import keras as keras
+from tensorflow.python.training.rmsprop import RMSPropOptimizer
+from tensorflow.python.platform import test
+from tensorflow.python.framework import test_util as tf_test_util
+
+
+def create_and_fit_Sequential_model(layer,shape):
+    #Helperfunction for quick evaluation
+    model=keras.models.Sequential()
+    model.add(layer)
+    model.add(keras.layers.Dense(32))
+    model.add(keras.layers.Dense(1))
+
+    model.compile(optimizer=RMSPropOptimizer(0.01),loss="categorical_crossentropy")
+    layer_shape=(10,)+shape
+    input_batch=np.random.rand(*layer_shape)
+    output_batch=np.random.rand(*(10,1))
+    model.fit(x=input_batch,y=output_batch, epochs=1, batch_size=1)
+    return model
+
+
+class normalization_test(test.TestCase):
```

Comment:
Could you copy LayerNormalizationTest in https://github.com/tensorflow/tensorflow/commit/cb40852a6ee804b12ab71aadb12babf96b090991 , and modify it to test our LayerNormalization here ?

And similarly, could you add InstanceNormalizationTest and GroupNormalizationTest ?